app/core/reservation.py

```diff
@@ -10,12 +10,12 @@ def __init__(self, room,holder,time,description,reservationId):
         self.reservationId = reservationId
 
     # Print method for debugging
-    def _print(self):
-        print("Reservation Info")
-        print("Holder: " + str(self.user.getName()))
-        self.time._print()
-        print("Description: " + str(self.description))
-        print("RID: " + str(self.reservationId))
+    def __str__(self):
+        return "Reservation Info" +\
+        "Holder: " + str(self.user.getName()) +\
+        self.time +\
```

Comment:
Observation, is self.time already a string at this point? I could not infer the type or find an originating instance, so I don't know what class this would be...